Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN go build .
 
 
 # Run binary
-FROM ubuntu:20.04
+FROM ubuntu:22.04
```

Comment:
We use to include version updates in the changelog. Consider also mentioning changes in the Go runtime that may affect the registry, if any.